Sources_v3/Database/DTOs/UserDTO_Tests.swift

```diff
@@ -174,4 +174,116 @@ class UserDTO_Tests: XCTestCase {
         // Check the online status is `false`
         XCTAssertEqual(database.viewContext.user(id: userId)?.isOnline, false)
     }
+    
+    func test_userWithUserListQuery_isSavedAndLoaded() {
+        let query = UserListQuery(filter: .autocomplete("name", with: "a"))
```

Comment:
nit: I know it's a stupid comment, but because we decided to implement search separately, can you use a different example for filter than `autocomplete?` It can become confusing in the future

Reply:
Good comment! I thought about it at the last moment but decided that it's fine.